### DISCUSSION TITLE
v0.62.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#3190] [Announcements]
### Enhancements

* Updated navigation sidebar design with ability to collapse and expand collections that are not active
* Added navigation sidebar to shared documents https://github.com/outline/outline/pull/2899
* Added reordering to starred documents https://github.com/outline/outline/pull/2953
* Support SSL without usage of reverse proxy https://github.com/outline/outline/pull/2959
* Display progress of import operations in settings https://github.com/outline/outline/pull/3064
* Added the ability to choose a collection as default home https://github.com/outline/outline/pull/3029
* Added DBdiagram (dbdiagram.io) embed by @n3n in https://github.com/outline/outline/pull/3124
* Show icon on external links https://github.com/outline/outline/pull/3100
* Add "new doc" button on collections in sidebar https://github.com/outline/outline/pull/3174
* Add "danger" background to dangerous menu items on hover https://github.com/outline/outline/pull/3148

#### Editor

* fix: Templates menu does not appear when collaborative editing is enabled https://github.com/outline/outline/pull/2915
* Editor shortcut conflicts https://github.com/outline/outline/pull/2943
* Emoji in title positioning https://github.com/outline/outline/pull/2927
* fix: changing the title and body content in quick succession would trigger unsaved changes warning prompt https://github.com/outline/outline/pull/2950
* fix: Flash of content when selecting text on RHS of document https://github.com/outline/outline/pull/2981
* fix: remove scrollbar on x-axis in Table of Contents by @iamsaumya in https://github.com/outline/outline/pull/3008
* fix: Ensure copy code button appears in collaborative editing mode https://github.com/outline/outline/pull/3021
* fix: Disabling editor embeds should work with collaborative editing https://github.com/outline/outline/pull/2968
* fix: Do not parse `:smile:` style emoji in Markdown paste handler https://github.com/outline/outline/pull/3056
* fix: Improvements to image positioning https://github.com/outline/outline/pull/3061

#### Platform

* fix: Missing space character by @rossmeissl in https://github.com/outline/outline/pull/2961
* fix: trim collection name on save rather than on change by @thenanyu in https://github.com/outline/outline/pull/2962
* fix: truncate a very long team name with ellipses by @thenanyu in https://github.com/outline/outline/pull/2963
* fix: Share popover incorrectly displays draft as publicly shared https://github.com/outline/outline/pull/2982
* fix: consistently check allowed domains by @eug-vs in https://github.com/outline/outline/pull/2985
* fix: issue-2974 reparent nested document when dropped on current collection by @ChuckJonas in https://github.com/outline/outline/pull/2975
* fix: Should be able to unstar archived and trashed documents https://github.com/outline/outline/pull/2983
* fix: Team logo shows as white in settings https://github.com/outline/outline/pull/3015
* fix: Cannot view history with view-only permissions and collaborative editing enabled https://github.com/outline/outline/pull/3024
* fix: Added Table of Contents to mobile views and account for branding on shared view layouts by @thenanyu in https://github.com/outline/outline/pull/2997
* fix: error from api rate limiting by @iamsaumya in https://github.com/outline/outline/pull/3050
* fix: make mobile popover dialog styling nice by @thenanyu in https://github.com/outline/outline/pull/3059
* fix: Images no longer in original document not included in export https://github.com/outline/outline/pull/3063
* fix: Auth persistence to `localStorage` https://github.com/outline/outline/pull/3078
* fix: Export now includes all documents that have identical names https://github.com/outline/outline/pull/3098
* fix: sync the correct collection with edit action by @iamsaumya in https://github.com/outline/outline/pull/3166
* fix: Reuse `InputSearch` style for move dialog https://github.com/outline/outline/pull/3173
* fix: editing collections should not forward to collection on save by @iamsaumya in https://github.com/outline/outline/pull/3187

## New Contributors
* @max-mykhailenko made their first contribution in https://github.com/outline/outline/pull/2895
* @rossmeissl made their first contribution in https://github.com/outline/outline/pull/2961
* @eug-vs made their first contribution in https://github.com/outline/outline/pull/2985
* @ChuckJonas made their first contribution in https://github.com/outline/outline/pull/2975
* @louisphn made their first contribution in https://github.com/outline/outline/pull/3034

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.62.0'>v0.62.0</a>.</em>

**Comment by max-mykhailenko** [comment]
@tommoor looks like i have a problem with DB, maybe we need some migrations?
```
original: error: column team.defaultCollectionId does not exist
      at Parser.parseErrorMessage (/opt/outline/node_modules/pg-protocol/dist/parser.js:278:15)
      at Parser.handlePacket (/opt/outline/node_modules/pg-protocol/dist/parser.js:126:29)
```

**Reply by max-mykhailenko** [reply]
Yarn migrate command doesn't apply migrations where this field was added. I use latest docker image

**Reply by max-mykhailenko** [reply]
Sorry, my fault, migrations helps
